ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/components/properties/FontSpec.kt

```diff
@@ -58,9 +60,50 @@ internal sealed interface FontSpec {
 internal fun Map<FontAlias, FontsConfig>.determineFontSpecs(
     resourceProvider: ResourceProvider,
 ): Map<FontAlias, FontSpec> {
+    val resourceFontFamilies = values
+        .toSet()
+        .map { it.android }
+        .filterIsInstance<FontInfo.Name>()
+        .mapNotNull { fontInfo -> fontInfo.family?.let { family -> fontInfo to family } }
+        .groupBy({ it.second }, { it.first })
```

Comment:
This is a way to group by non-null family properties while making the compiler happy and avoiding the force-unwrap... Lmk if you have other suggestions, maybe a force-unwrap isn't so bad if we previously verify that the values is not null though...